src/frontend/src/lib/components/transactions/AllTransactionsList.svelte

```diff
@@ -9,9 +9,13 @@
 	import { mapAllTransactionsUi } from '$lib/utils/transactions.utils';
 
 	let transactions: AllTransactionsUi;
+	// TODO: add ethTransactions to mapAllTransactionsUi
```

Comment:
and `ckEthMinterInfo` I guess?

Speaking of: are we sure? If I'm not wrong those info are currently not loaded on the tokens page, this will add some more stress.

Reply:
corrected!

> are we sure?

Good catch! I thought it was among the `Loaders`, but indeed it is specific to Ethereum transaction page (via the `ConvertToCkETH` button) or in IC transactions page (via `ConvertToEth` button and component `IcTransactionsCkEthereumListeners`). I will add a listener in another PR, then, good for you?